config.md

```diff
@@ -79,8 +79,70 @@ varnish:
 ```
 and the rest will be merged from the defaults.
 
+## Using Wundersecrets
+You can use shared secret variables by providing a private repository into `conf/project.yml`. The repository needs to contain `ansible.yml` in the root folder. Variables from that file are are to wundertools run automatically.
```

Comment:
This sentence needs some work:
"Variables from that file are are to wundertools run automatically."